src/components/atoms/Icon/index.jsx

```diff
@@ -0,0 +1,3 @@
+import Icon from './Icon';
```

Comment:
suggestion: We can reduce the two line to:

```suggestion
export { default } from './Icon';
```